cmd/command.go

```diff
@@ -594,6 +595,34 @@ func hIncrBy(cli *Client, args [][]byte) (interface{}, error) {
 	return cli.db.HIncrBy(key, field, incrInt64Val)
 }
 
+func hRandField(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) != 1 || len(args) != 2 || len(args) != 3 {
+		return nil, newWrongNumOfArgsError("hrandfield")
+	}
+	if len(args) == 1 {
+		keys, err := cli.db.HRandField(args[0], 1, false)
+		if err != nil {
+			return nil, err
+		}
+		if len(keys) == 1 {
+			return keys[0], nil
+		}
+		return keys, nil
+	}
+	count, err := strconv.Atoi(string(args[1]))
+	if err != nil {
+		return nil, errValueIsInvalid
+	}
+	withValues := false
+	if len(args) == 3 {
+		if strings.ToUpper(string(args[2])) != argsWithValues {
```

Comment:
You can use [`strings.EqualFold()`](https://pkg.go.dev/strings#EqualFold).

Reply:
Sorry for the late reply.

Thanks, I had never heard of `strings.EqualFold()` before.
I will fix it.